Gui/opensim/visualizationServer/src/org/eclipse/jetty/JettyMain.java

```diff
@@ -58,8 +69,9 @@ public static void main(String[] args) {
     public static void startServer() {
         try {
             if (!serverup){
+                getSERVER_PORT(); // set port based on command line if any
```

Comment:
From what I can tell, `getSERVER_PORT()` does not edit the `SERVER_PORT` static variable.

Reply:
The static block in JettyMain is what's responsible for parsing the command line option and setting the static variable SERVER_PORT